src/main/java/edu/ie3/datamodel/utils/validation/ConnectorValidationUtils.java

```diff
@@ -80,7 +80,7 @@ private static void checkLine(LineInput line) {
   /**
    * Validates a line type if: <br>
    * - it is not null <br>
-   * - B is greater/equal to 0 (Phase-to-ground susceptance per length) <br>
+   * - B is less than 0 (Phase-to-ground susceptance per length) <br>
```

Comment:
Sorry for the flaw in issue description. It is fine, if the succeptance is zero. So negative values + zero shall pass.

Reply:
Made the required changes